docker/scripts/common.sh

```diff
@@ -71,6 +71,11 @@ e2e_test_relayer() {
     local privkey=$4
     local txhash=$5
     local relayer=${root}/relayer/relayer-node18-linux-x64
+
+    unamestr=$(uname)
+    if [[ "$unamestr" == 'Darwin' ]]; then
+       relayer=${root}/relayer/relayer-node18-macos-x64
+    fi
```

Comment:
Could you please make a new function, e.g. `get_relayer_binary` that returns that and use the function here as well in other places that need it:
```
docker/optimism/optimism.sh:25:    local relayer=${ROOT}/relayer/relayer-node18-linux-x64
docker/optimism/optimism.sh:79:    relayer=${ROOT}/relayer/relayer-node18-linux-x64
docker/polygon_zkevm/polygon_zkevm.sh:69:        relayer=${ROOT}/relayer/relayer-node18-linux-x64
docker/polygon_zkevm/polygon_zkevm.sh:71:        relayer=${ROOT}/relayer/relayer-node18-macos-x64
```
?